actions/playerActions.js

```diff
@@ -29,15 +30,18 @@ const getPlayerError = (payload) => ({
 export const getPlayer = (accountId, host = HOST_URL) => (dispatch) => {
   dispatch(getPlayerRequest());
   dispatch(getPlayerMatchesRequest());
+  dispatch(getPlayerHeroesRequest());
```

Comment:
is this making another request?  The /players endpoint is supposed to return both matches and heroes.